### PR TITLE
Fixed search with pg_search gem

### PR DESCRIPTION
**Pull request by majioa**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -5,7 +5,8 @@ def title(page_title)
     content_for(:title) { page_title.html_safe }
   end
 
-  def keywords(string)
+  def keywords string
+    string ||= ''
```
Comment (biow0lf) [inline app/helpers/application_helper.rb]: Rollback this change

File: app/models/srpm.rb [hunk 2]
```diff
@@ -39,8 +39,13 @@ class Srpm < ApplicationRecord
            primary_key: 'name',
            foreign_key: 'repo' # dependent: :destroy
 
-  validates :groupname, presence: true
+  multisearchable against: [:name, :summary, :description, :filename, :url]
+
+  pg_search_scope :query,
+                  against: %i(name summary description filename url),
+                  using: { tsearch: { prefix: true}}
```
Comment (biow0lf) [inline app/models/srpm.rb]: Move this after `after_destroy` callback.

Use `[]` for arrays.

**Comment by biow0lf** [comment]
And, how about speed? How fast this new search? Maybe, we are need indexes?

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (majioa) [reply app/helpers/application_helper.rb]: avoiding this breaks my run

File: app/models/srpm.rb (hunk 2 above)
Comment (majioa) [reply app/models/srpm.rb]: should `pg_scopes` be with the scope group, or be in another place apart it?

**Comment by majioa** [comment]
addressed.

I'havent measured speed yet.

**Comment by biow0lf** [comment]
Deployed. [openbox example](https://packages.altlinux.org/uk/search?utf8=✓&branch=Sisyphus&query=openbox). Looks like search ignore branch. @majioa

**Comment by biow0lf** [comment]
And new search is crazy slow.

**Pull request merged**